cl_dll/StudioModelRenderer.h

```diff
@@ -157,6 +151,15 @@ class CStudioModelRenderer
 	// Pointer to header block for studio model data
 	studiohdr_t		*m_pStudioHeader;
 	
+	// Change viewmodel FOV
+	cvar_t			*m_pCvarViewmodelFov;
+
+	// Disable viewmodel idle/fidget animations on viewmodels
+	cvar_t			*m_pCvarViewmodelNoIdle;
+
+	// Disable viewmodel draw/holster/deploy animations on viewmodels
+	cvar_t			*m_pCvarViewmodelNoEquip;
+
```

Comment:
Might as well move them all the way down to prevent any and all future issues.

Reply:
Done.